packages/salesforcedx-vscode-core/src/settings/sfdxCoreSettings.ts

```diff
@@ -73,6 +74,10 @@ export class SfdxCoreSettings {
     return this.getConfigValue(PREFER_DEPLOY_ON_SAVE_ENABLED, false);
   }
 
+  public getEnableSourceTrackingForDeployAndRetrieve(): boolean {
+    return this.getConfigValue(ENABLE_SOURCE_TRACKING_FOR_DEPLOY_RETRIEVE, false);
+  }
```

Comment:
Do we have a jest unit test suite for this file?

Reply:
No we don't have any jest file for this.

Reply:
Lets start that file and get one unit test in it! :)